arex-instrumentation-api/src/main/java/io/arex/inst/runtime/serializer/Serializer.java

```diff
@@ -60,6 +60,30 @@ public static void initSerializerConfigMap() {
         }
     }
 
+    public static String serializeWithType(Object object) {
+        if (object == null || INSTANCE == null) {
+            return null;
+        }
+        try {
+            return INSTANCE.getSerializer(ArexConstants.JACKSON_SERIALIZER_WITH_TYPE).serialize(object);
+        }catch (Throwable ex) {
+            LogManager.warn("serializer-serialize-with-type", StringUtil.format("can not serialize object: %s, cause: %s", TypeUtil.errorSerializeToString(object), ex.toString()));
+            return null;
+        }
```

Comment:
Suggest call `serialize` method directly to reduce duplicate code.
`return serialize(object, ArexConstants.JACKSON_SERIALIZER_WITH_TYPE);`

Reply:
`public static String serialize(Object object, String serializer) {`
Suggest print `serializer` to log the log